chart/kubeapps/values.yaml

```diff
@@ -1818,9 +1818,12 @@ redis:
       ## allkeys-lru: evict keys by trying to remove the less recently used (LRU) keys first, in order 
       ## to make space for the new data added
       - "--maxmemory-policy allkeys-lru"
+      ## ref https://stackoverflow.com/questions/22815364/flushall-and-flushdb-commands-on-redis-return-unk-command
+      ## Redis official Helm chart by default disables FLUSHDB and FLUSHALL commands
+    disableCommands: []
   replica:
     ## @param redis.replica.replicaCount Number of Redis&trade; replicas to deploy
-    replicaCount: 0
+    replicaCount: 1
```

Comment:
Hmm, did we use 0 by default before we had the enable/disable switch? If so, +1, and should it default to 2?

Reply:
looks that way. I was testing with 1 for now. Can switch to 2 later